core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerRpcClientBuilder.java

```diff
@@ -36,12 +36,24 @@ public CircuitBreakerRpcClientBuilder(CircuitBreakerStrategyWithContent<RpcRespo
     }
 
     @Override
-    CircuitBreakerRpcClient build(Client<RpcRequest, RpcResponse> delegate) {
-        return new CircuitBreakerRpcClient(delegate, circuitBreakerMapping(), strategyWithContent());
+    public CircuitBreakerRpcClient build(Client<RpcRequest, RpcResponse> delegate) {
+        return new CircuitBreakerRpcClient(delegate, mapping(), strategyWithContent());
     }
 
     @Override
-    Function<Client<RpcRequest, RpcResponse>, CircuitBreakerRpcClient> newDecorator() {
+    public Function<Client<RpcRequest, RpcResponse>, CircuitBreakerRpcClient> newDecorator() {
         return this::build;
     }
+
+    // Methods that were overridden to change the return type.
+
+    @Override
+    public CircuitBreakerRpcClientBuilder circuitBreakerMapping(CircuitBreakerMapping mapping) {
+        return (CircuitBreakerRpcClientBuilder) super.circuitBreakerMapping(mapping);
```

Comment:
`super.circuitBreakerMapping(mapping);` -> `super.mapping(mapping);`?

Reply:
Just replaced with `mapping(mapping)` which is simpler.